runtime/autoload/ccomplete.vim

```diff
@@ -2,9 +2,10 @@ vim9script noclear
 
 # Vim completion script
 # Language:     C
-# Maintainer:   Bram Moolenaar <Bram@vim.org>
+# Maintainer: The Vim Project <https://github.com/vim/vim>
+# Former Maintainer:   Bram Moolenaar <Bram@vim.org>
```

Comment:
```suggestion
# Maintainer:   The Vim Project <https://github.com/vim/vim>
# Former Maintainer: Bram Moolenaar <Bram@vim.org>
```
How about aligning "Language:", "Maintainer:" and "Last Change:"?
Not sure if "Former Maintainer:" should be also aligned.

Reply:
okay, will adjust later.